tools/rna_tools/intarna/intarna.xml

```diff
@@ -258,20 +280,29 @@
                     <param argument="--seedTRange" type="text" value=""
                         label="Interval(s) in the target to search for seeds" help="in format 'from1-to1,from2-to2,...' (Note, only for single query)"/>
                 </section>
-                <section name="interaction" title="Interaction Parameters">
+                <section name="shape" title="SHAPE Input Parameters">
+                    <param argument="--qShape" format="txt" type="data" label="SHAPE reactivity data to guide the query's accessibility computation" help="Two column text file with sequence positions (1-based) and normalized reactivity values. Missing values may be left out, or assigned a negative score" />
+                    <param argument="--tShape" format="txt" type="data" label="SHAPE reactivity data to guide the query's accessibility computation" help="Two column text file with sequence positions (1-based) and normalized reactivity values. Missing values may be left out, or assigned a negative score" />
+                </section>
+                <section name="interaction" title="Prediction Parameters">
                     <expand macro="interaction_macro"/>
                     <param argument="--pred" type="select" label="Prediction target" help="Sets what to optimize for">
                         <option value="S" selected="true">Single-site minimum-free-energy interaction</option>
-                        <option value="P">Single-site maximum-probability interaction</option>
+                        <!-- <option value="P">Single-site maximum-probability interaction</option> -->
                     </param>
                     <param argument="--energy" type="select" label="Energy computation" help="">
                         <!-- <option value="B">Base pair == -1 ()</option> -->
                         <option value="V" selected="true">VRNA-based computation</option>
                     </param>
-                    <param argument="--energyVRNA" type="data" format="txt" optional="true"
+                    <!-- <param argument="--energyVRNA" type="data" format="txt" optional="true"
```

Comment:
the `--` inside the comment make the comment invalid :(

Reply:
removed